src/main/java/crazypants/enderio/machine/AbstractPoweredMachineEntity.java

```diff
@@ -183,8 +179,15 @@ private void updateCapacitorFromSlot() {
   @Override
   public void readCommon(NBTTagCompound nbtRoot) {
     super.readCommon(nbtRoot);
-    handleCapacitorLoading(nbtRoot);
-    handleStoredEnergyLoading(nbtRoot);
+    setCapacitor(Capacitors.values()[nbtRoot.getShort("capacitorType")]);
+    int energy;
+    if (nbtRoot.hasKey("storedEnergy")) {
+      float storedEnergyMJ = nbtRoot.getFloat("storedEnergy");
```

Comment:
I would suggest extracting strings into constants (`storedEnergy`, `capacitorType`) so it's a compile time error if someone has a typo while adding new features/refactoring.

Reply:
Hmm, none of the other NBT storing methods in Ender IO seem to use string constants, so it's probably better to be consistent? This code is also actually (intentionally) identical to what upstream has:
https://github.com/SleepyTrousers/EnderIO/blob/release/1.7.10/2.3/src/main/java/crazypants/enderio/machine/AbstractPoweredMachineEntity.java#L159-L184

The constants are also only referenced within these two methods (and this should not generally change), so we're probably okay since the references are localized.